content/tutorials/getting-started.md

```diff
@@ -78,8 +79,19 @@ This starts a KES server on `127.0.0.1:7373` and stores keys in memory.
      export PATH=$PATH:@HOME/minio-binaries/
      ```
 
-   **Important:** Invoke the Windows executable file from the terminal, PowerShell, or Command Prompt.
-   You cannot double click the file from the Windows graphical user interface.
+   **Important:** 
+   
+   - Windows: Invoke the Windows executable file from the terminal, PowerShell, or Command Prompt.
+     You cannot double click the file from the Windows graphical user interface.
+
+   - MacOS: Create an exception to allow MacOS to open the executable downloaded from the Internet.
```

Comment:
```suggestion
   - For MacOS: Create an exception to allow MacOS to open the executable downloaded from the Internet.
```

Reply:
Technically the current product name is styled `macOS` but I'm not sure if it's worth hunting down all these right now.

Reply:
Apple trademark lawyers thank you very much!